frontend/src/app/workflows/mockup-viewer/mockup-viewer.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="h-100">
+  <div class="row">
+    <div class="col align-self-start">
+      <button type="button" class="btn btn-secondary btn-sm m-1"
+              (click)="setFirst()"
+              [disabled]="currentDisplay === 'class'">
+        First
+      </button>
+      <button type="button" class="btn btn-secondary btn-sm m-1"
+              (click)="previous()"
+              [disabled]="currentDisplay === 'class'">
+        Previous
+
+      </button>
+      <button type="button" class="btn btn-secondary btn-sm m-1"
+              (click)="next()"
+              [disabled]="currentDisplay === 'class'">
+        Next
+      </button>
+      <button type="button" class="btn btn-secondary btn-sm m-1"
+              (click)="setLast()"
+              [disabled]="currentDisplay === 'class'">
+        Last
+      </button>
+    </div>
+
+    <div class="mr-3">
+      <div class="btn-group btn-group-toggle" ngbRadioGroup name="radioBasic" [(ngModel)]="currentDisplay">
+        <label ngbButtonLabel class="btn btn-info btn-sm mt-1 mb-1">
+          <input ngbButton type="radio" [value]="'pages'"> Pages
+        </label>
+        <label ngbButtonLabel class="btn btn-info btn-sm mt-1 mb-1">
+          <input ngbButton type="radio" [value]="'objects'"> Objects
+        </label>
+        <label ngbButtonLabel class="btn btn-info btn-sm mt-1 mb-1">
+          <input ngbButton type="radio" [value]="'class'"> Class
+        </label>
```

Comment:
btn class ist unnecessary with ngbButtonLabel. Why do you add mt-1 and mb-1?

Reply:
I wanted it to have a little spacing obove and below the actual button. Similiar to the buttons on the left side for prvious/next... buttons.
And becuase it is a radioGroup i didn't want to have spacing between the buttons of the group.

Reply:
I worked out some new spacing for the Scenarios Editor Toolbar and I'd like to apply it here to make it consistent. I can make the changes later if you're ok with that.

Reply:
Yeah, sure.